cardano-mempool/README.md

```diff
@@ -0,0 +1 @@
+# primal-mempool
```

Comment:
This looks a bit spartan. Could use a little bit more information here.

Also, it looks like this is still the old project name.

Reply:
Renamed and added a little bit of explanation what this project is about